static/js/stats/stats.js

```diff
@@ -82,12 +82,96 @@ $(document).ready(function () {
     });
 });
 
-function populate_messages_sent_over_time(data) {
-    if (data.end_times.length === 0) {
-        // TODO: do something nicer here
-        return;
-    }
+function replace_data_with_fixture_data() {
```

Comment:
hm, is there anyway to do this with less code? I think you can use the re-layout of the regular (non-fixture data) charts.